src/GovUk.Education.ExploreEducationStatistics.Publisher/Services/ContentService.cs

```diff
@@ -43,17 +43,20 @@ public async Task DeletePreviousVersionsContent(params Guid[] releaseIds)
 
             foreach (var release in releases)
             {
-                await _fastTrackService.DeleteAllFastTracksByRelease(release.PreviousVersionId);
+                if (release.PreviousVersionId.HasValue)
+                {
+                    await _fastTrackService.DeleteAllFastTracksByRelease(release.PreviousVersionId.Value);
+                }
 
                 // Delete content which hasn't been overwritten because the Slug has changed
-                if (release.Slug != release.PreviousVersion.Slug)
+                if (release.PreviousVersion != null && release.Slug != release.PreviousVersion.Slug)
```

Comment:
You could move this within the `if` above since that's also checking whether a Previous Version exists

Reply:
Cool done